core/sup_tree_gear/gear_log_writer.ex

```diff
@@ -109,6 +109,13 @@ defmodule AntikytheraCore.GearLog.Writer do
     end
   end
 
+  defun rotate(gear_name :: v[GearName.t]) :: :ok do
```

Comment:
`send(name, :rotate)` returns `:rotate`.

```suggestion
  defun rotate(gear_name :: v[GearName.t]) :: :ok | :rotate do
```

Reply:
Thank you! I referred to wrong documentation.

I don't think `:ok | :rotate` is a useful return value. `:ok | :err` may be better.

Reply:
Finally, I changed the function `rotate` to return `:ok` just like `set_min_level`.
20bd08aa2984916c8c4390d8618242b44350ad69
(I also wanted to change `rotate_and_start_upload_in_all_nodes`, but gave up because the test needed to be fixed.)